### DISCUSSION TITLE
Arm64 Support

### DISCUSSION THREAD

**Discussion by RedstoneWizard08** [forem/forem#16765] [Feature Requests]
### Description
Provide support for arm64 CPUs to run the Docker image of Forem. This will include the Mac M1 chips, general aarch64 chips (`linux/arm64`), and Raspberry Pis. This will also continue to allow amd64 chips to run it as well.

### Potential Steps
Update Forem ruby image for arm64 and amd64 (and please make it open-source), Dockerfile is below.
Possible Forem AIO image (but use the regular one, it's better, just add arm64 support)
Update Forem docker image to allow for arm64 and amd64 architectures.

### Notes
- To determine the architecture, use this kind of thing in a Dockerfile:
```dockerfile
RUN export ARCH=$(uname -m)
```
- Stuff is still WIP.

### Files
See here: https://github.com/RedstoneWizard08/Forem-Multiarch-Docker

**Comment by klardotsh** [comment]
Converting this to an issue now that this is actively being worked on: https://github.com/forem/forem/issues/19626
